implementations/rust/ockam/ockam_api/tests/journeys.rs

```diff
@@ -40,7 +40,7 @@ fn test_create_journey_event() {
                 let cli = CliState::create(cli_state_directory)
                     .await
                     .unwrap()
-                    .set_tracing_enabled();
+                    .set_tracing_enabled(true);
```

Comment:
I'd rather have an `enable_tracing()/disable_tracing()` pair of functions, but I'm not sure we are following that pattern anywhere in the codebase.

Reply:
This is useful because we pass a boolean in `CommandGlobalOpts`:
```rust
 state.set_tracing_enabled(tracing_configuration.is_enabled()),
```
Otherwise we would have to surround with a more verbose `if/else` expression.